src/ripple/protocol/AnyPublicKey.h

```diff
@@ -102,9 +102,16 @@ class AnyPublicKey
 
 public:
     AnyPublicKey() = delete;
-    AnyPublicKey (AnyPublicKey const&) = delete;
     AnyPublicKey& operator= (AnyPublicKey const&) = delete;
 
+    AnyPublicKey (AnyPublicKey const& other)
+        : buffer_type(other.buffer_type::member.data(),
+            other.buffer_type::member.size())
+        , AnyPublicKeySlice (buffer_type::member.data(),
+            buffer_type::member.size())
+    {
+    }
```

Comment:
We definitely don't want to store a slice in `UniqueNodeList`. I think this is sufficient evidence that we really do need a copy constructor (because we have to put it in two places).